formats/html_publication/frontend/examples/prime_ministers_office.json

```diff
@@ -5,7 +5,9 @@
     "headings": "<ol></ol>",
     "public_timestamp": "2016-01-17T14:19:42.460Z",
-    "first_published_version": false
+    "first_published_version": false,
+    "isbn": "978-1-78386-459-1",
```

Comment:
The example content items should reflect the live data? This looks like isbn and the contact address have been copied and pasted to each example.